applications/launchpad/gui-react/src/components/ProgressBar/index.tsx

```diff
@@ -0,0 +1,52 @@
+import { useEffect, useRef, useState } from 'react'
+import { config, useSpring } from 'react-spring'
+import Text from '../Text'
+import { Track, Fill, StyledProgressBar, Tip } from './styles'
+
+import { ProgressBarProps } from './types'
+
+const limitValue = (value: number) => {
+  return value > 100 ? 100 : Math.abs(value)
```

Comment:
just a tip: you can do things like this with `Math.min(100, Math.abs(value))`

I don't think there's any actual difference in the speed or amount of operations, but to me it looks nicer xD